include/aspect/mesh_deformation/fastscapecc_adapter.h

```diff
@@ -0,0 +1,234 @@
+#ifndef FASTSCAPELIB_DEALII_GRID_H_
+#define FASTSCAPELIB_DEALII_GRID_H_
+
+#include <unordered_map>
+#include <vector>
+
+#include <xtensor/xbroadcast.hpp>
+#include <xtensor/xmath.hpp>
```

Comment:
In my installation, these files live at `xtensor/views/xbroadcast.hpp` and `xtensor/core/xmath.hpp`. Is this a version issue where xtensor has moved files around? How do we want to address this?